packages/fxa-auth-server/test/oauth/api.js

```diff
@@ -3532,6 +3532,7 @@ describe('/v1', function () {
             refresh_token_id: tokenId,
           }),
         });
+        console.log(res.result);
```

Comment:
Did you mean to leave this in here?

Reply:
nope, thanks :)

Reply:
IIRC we used to have linting for this, did it get removed?